core/src/main/resources/reference.conf

```diff
@@ -170,12 +170,10 @@ kanaloa {
     # Metrics report configuration
     metrics {
       enabled = off
-      #  statsd {
-      #     host = "localhost"
+      #  use statsD for this metrics, also need to config statsD {host and port } down below
+      #  statsD {
```

Comment:
I think this can be addressed when we break out the statsD stuff into its own package.  We can address the defaults/overriding structure and the colocation of these statsD parameters into a single config block